File: spinner.go
package main

import "bufio"
import "fmt"
import "os"
import "time"

func main() {
	STDOUT := bufio.NewWriter(os.Stdout)
	humanoids := []string{"\\o/", "\\o>", "<o>", "<o/"}

	for {
		for _, humanoid := range humanoids {
			fmt.Fprint(STDOUT, "\r", humanoid)
			STDOUT.Flush()
			time.Sleep(100 * time.Millisecond)
		}
	}
}

File: spinner.rs
use std::{thread, time};
use std::io::{self, Write};


fn main() {
    let humanoids = ["\\o/", "\\o>", "<o>", "<o/"];
    let pause = time::Duration::from_millis(100);
    let mut stdout = io::stdout();
    
    loop {
        for humanoid in humanoids {
            print!("\r{}", humanoid);
            stdout.flush().unwrap();
            thread::sleep(pause);
        }
    }
}
